spk/synocli-file/Makefile

```
@@ -1,37 +1,30 @@
 SPK_NAME = synocli-file
 SPK_VERS = 1.0
```

I propose to increase package version itself too
```suggestion
SPK_VERS = 1.1
```

---

My intent is to update revision for updated components and update the version with adding/removing components.
But we could use major/minor package version for this.

---

I prefer the other way "major/minor" to get a reason to increase major from time to time...